lib/yast/index_servlet.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+#--
+# Yast rake
+#
+# Copyright (C) 2020, SUSE LLC
+#   This library is free software; you can redistribute it and/or modify
+# it only under the terms of version 2.1 of the GNU Lesser General Public
+# License as published by the Free Software Foundation.
+#
+#   This library is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+# details.
+#
+#   You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+#++
+
+require "webrick"
```

Comment:
is it part of standard ruby? Otherwise it will crash.

Reply:
Yes, webrick is part of the [Ruby stdlib](https://ruby-doc.org/stdlib-2.5.0/libdoc/webrick/rdoc/index.html).

Reply:
```console
$ rpm -qf $(ruby -rwebrick -e 'puts $LOADED_FEATURES.grep(/webrick/)') | uniq
ruby2.5-stdlib-2.5.5-lp151.4.3.1.x86_64
```
Q.E.D.